# WaypointInformation.msg
string waypoint_name
float64 x
float64 y
float64 theta